back/engines/commercial/verification/app/services/verification/patches/permissions_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Verification
+  module Patches
+    module PermissionsService
+      def denied_when_permitted_by_groups?(permission, user)
+        denied_reason = super
+
+        if denied_reason == :not_permitted && !user.verified?
+          permission.groups.each do |group|
```

Comment:
User `VerificationService.new.find_verification_group(groups)`

Reply:
Quick question though, what did you do with this?